tests/test_optimizer_random_dag.py

```diff
@@ -6,8 +6,10 @@
 
 import sky
 from sky import clouds
+from sky.clouds import service_catalog
 
-ALL_INSTANCE_TYPES = sum(sky.list_accelerators(gpus_only=True).values(), [])
+ALL_INSTANCE_TYPE_INFOS = sum(
+    sky.list_accelerators(gpus_only=True).values(), [])
 GCP_HOST_VM = 'n1-highmem-8'
```

Comment:
Shall we delete this line since the variable is not used?

Reply:
Good catch! Removed. Thanks!